src/lib/template/msg_tsd.hbs

```diff
@@ -16,3 +16,6 @@
 {{#each enums}}
 {{{render 'partial/enum' this}}}
 {{/each}}
+{{#each wellKnownDeclarations}}
+{{{this}}}
```

Comment:
I think we need `partial/xxx` sub templates for well known types

Reply:
Cool, I'll wait to hear back on whether you prefer a sub template setup from elsewhere before implementing just so I know we're on the same page.